src/debugger.ts

```diff
@@ -21,43 +17,51 @@ import { ISignal, Signal } from '@phosphor/signaling';
 
 import { SplitPanel } from '@phosphor/widgets';
 
-import { IObservableString } from '@jupyterlab/observables';
+import { Breakpoints } from './breakpoints';
+
+import { Callstack } from './callstack';
+
+import { DebuggerEditors } from './editors';
+
+import { DebugService } from './service';
 
 import { IDebugger } from './tokens';
 
-import { IDataConnector } from '@jupyterlab/coreutils';
+import { Variables } from './variables';
 
 export class Debugger extends SplitPanel {
   constructor(options: Debugger.IOptions) {
     super({ orientation: 'horizontal' });
     this.title.label = 'Debugger';
     this.title.iconClass = 'jp-BugIcon';
 
-    this.model = new Debugger.Model(options);
+    this.model = new Debugger.Model({
+      connector: options.connector
```

Comment:
Not related to this change, but I'm wondering if we actually still need the connector here.

The `DebugSession` is the one acting as the connector when comparing it to the JupyterLab `inspector` extension.

Reply:
We can track that in a separate issue though.

Reply:
Opened https://github.com/jupyterlab/debugger/issues/104